src/paths.cpp

```diff
@@ -66,6 +66,7 @@ wxString GetConfigFile()
 //     wxStandardPaths::GetResourcesDir
 //     wxStandardPaths::GetAppDir
 // Fortunately these should always return the exe directory.
+// TODO: Investigate using wxStandardPaths::DontIgnoreAppSubDir with wxWidgets 2.9.1+
 #if defined(__WXDEBUG__) && defined(__WXMSW__)
```

Comment:
That's fine -- this is getting to be a big PR.  I just have a couple things I want to test out to make sure they still work on windows.